ci-operator/step-registry/cucushift/installer/check/ibmcloud/cucushift-installer-check-ibmcloud-commands.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+
+function run_command() {
+    local CMD="$1"
+    echo "Running Command: ${CMD}"
+    eval "${CMD}"
+}
+
+#OCP-47390 - [IPI-on-IBMCloud] Install cluster with "install-config.yaml" last step
+function checkProviderType {
+    local platformStatusFile="${ARTIFACT_DIR}/platformStatus"
+    oc get infrastructure/cluster -o yaml > ${platformStatusFile}
+    yq-go r ${platformStatusFile} 'status.platformStatus'
+    providerType=$(yq-go r ${platformStatusFile} 'status.platformStatus.ibmcloud.providerType')
+    if [[ ${providerType} != "VPC" ]]; then
+        echo "ERROR: ${providerType} is not expected providerType [VPC]!!"
+        return 1
+    else 
+        return 0
+    fi
+}
+
+#OCP-47395 - [IPI-on-IBMCloud] Install cluster with realtime (RT) kernel
+function checkEnabledRealtimeKernel {
```

Comment:
I think we can prompt this check to be a common installer check step, so that other platform also can reuse it.

Reply:
How about add cucushift-installer-check-realtime step under ci-operator/step-registry/cucushift/installer/check/realtime?

Reply:
ok for me.